metrics/confusion_matrix/confusion_matrix.py

```diff
@@ -41,6 +42,13 @@
         >>> results = confusion_matrix_metric.compute(references=[0, 1, 2, 0, 1, 2], predictions=[0, 1, 1, 2, 1, 0])
         >>> print(results)  # doctest: +ELLIPSIS +NORMALIZE_WHITESPACE
         {'confusion_matrix': array([[1, 0, 1], [0, 2, 0], [1, 1, 0]][...])}
+
+    Example 2-Multilabel scenario
+        >>> you must pass (config_name="multilabel") to the load method
```

Comment:
This code is being executed in the tests, so this is causing the error as it's not valid syntax - see https://github.com/huggingface/evaluate/actions/runs/7488835245/job/20386909998?pr=533

```suggestion
        >>> # you must pass (config_name="multilabel") to the load method
```